Demo/Components/CarouselVIew/CarouselViewDemoCell.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import FinniversKit
+
+class CarouselViewDemoCell: CarouselViewCell {
+
+    var text: String? {
+        get { return label.text }
+        set { label.text = newValue }
+    }
+
+    private lazy var label: UILabel = {
+        let label = UILabel(frame: .zero)
+        label.font = .systemFont(ofSize: 48, weight: .bold)
+        label.textColor = .white
+        label.backgroundColor = UIColor(hue: 0.6, saturation: 0.3, brightness: 0.6, alpha: 1.0)
```

Comment:
Why don't you just use some system or FINN color here? Just asking 😄

Reply:
No, reason. I just like the colors with saturation 0.3 and brightness 0.6. I can use a FINN color here 😉